<?xml version="1.0"?>
<launch>
        <node pkg="drone_pose_estimation" type="distance_calculator_node" name="distance_calculator_node" output="screen">
            <!-- node parameters -->
            <param name="enable_console_output"  type="bool" value="true" />
            <param name="enable_rostopic_output"  type="bool" value="true" />
            <param name="enable_txt_file_output"  type="bool" value="true" />

            <param name="enable_stable_groundtruth"  type="bool" value="false" />
            <param name="stable_groundtruth_latitude"  type="double" value="28.2703016385"/>
            <param name="stable_groundtruth_longitude"  type="double" value="113.038538401"/>

            <param name="calculator_result_ros_topic"  type="string" value="/position_calculator/ground_state" />
            <param name="ground_truth_ros_topic"  type="string" value="/ground_01/dgps_node/dgps_fix_position" />
            <param name="distance_ros_topic"  type="string" value="distance" />

            <param name="target_id"  type="int" value="0" />
            
            <param name="txt_file_name"  type="string" value="/home/ubuntu/distance_txt.txt" />
        </node>
</launch>
